code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -643,11 +764,14 @@
 		qdel(O)
 	for (var/mob/M in bay.GetAllContents())
 		qdel(M)
+	for (var/turf/T in bay)
+		T.ChangeTurf(/turf/open/floor/plasteel)
```

Comment:
```suggestion
	for (var/turf/bay_turf in bay)
		bay_turf.ChangeTurf(/turf/open/floor/plasteel)
```

Reply:
Nope, but wouldn't i need it to be listed as `var/turf/bay_turf` in order to use the `ChangeTurf()` proc?